initPTB.m

```diff
@@ -152,3 +120,122 @@
 
 
 end
+
+
+function initDebug(cfg)
+
+% init PTB with different options in concordance to the debug Parameters
+Screen('Preference','SkipSyncTests', 0);
+if cfg.debug
+    
+    Screen('Preference', 'SkipSyncTests', 2);
+    Screen('Preference', 'Verbosity', 0);
+    Screen('Preferences', 'SuppressAllWarnings', 2);
+    
+    fprintf('\n\n\n\n')
+    fprintf('########################################\n')
+    fprintf('##   DEBUG MODE. TIMING WILL BE OFF.  ##\n')
+    fprintf('########################################')
+    fprintf('\n\n\n\n')
+    
+    testKeyboards(cfg)
+    
+end
+
+if cfg.testingTranspScreen
+    PsychDebugWindowConfiguration
+end
+
+end
+
+function initKeyboard
+
+% Make sure keyboard mapping is the same on all supported operating systems
+% Apple MacOS/X, MS-Windows and GNU/Linux:
+KbName('UnifyKeyNames');
+
+% Don't echo keypresses to Matlab window
+ListenChar(-1);
+
+end
+
+function cfg = initAudio(cfg)
+    
+    if cfg.initAudio
+        
+        requestedLatency = 3;
+        
+        InitializePsychSound(1);
+
+        cfg.audio.devIdx= [];
```

Comment:
all the parameters related to auditory stuff are stored in a field called `audio`.

this cfg structure is growing too big so we need to find a way to organize it.

Reply:
yes i am suffering from the same issue. But is it really an issue?if cfg has sub-structures e.g. for audio, data, visual, ... then it could be big and still easy to find things in it?

Reply:
no, but at least you might have a map where to find eg visual stuff related. Does it make sense? A long list of vars in cfg is as confusing, IMHO, and it does not spark joy

Reply:
OK I will merge this for now and open issues to remember to refactor the cfg structure: this will potentially "break" a lot of things so I would rather create some "tests" and templates first.